src/ucs/stats/stats.h

```diff
@@ -98,7 +85,6 @@ void ucs_stats_node_free(ucs_stats_node_t *node);
 #define UCS_STATS_ARG(_arg)
 #define UCS_STATS_RVAL(_rval) NULL
 #define UCS_STATS_NODE_DECLARE(_node)
-#define UCS_STATS_NODE_ALLOC(_p_node, _class, _parent, ...) UCS_OK
```

Comment:
or alternative: ```ucs_empty_function_return_success()```

Reply:
> or alternative: `ucs_empty_function_return_success()`

good idea
done, force-pushed new changes as the solution was completely changed